### PR TITLE
Improve essential travel navbar

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
- The essential travel overbrand was done in isolation and when introducing it to pattern library we noticed it doesn't work quite as expected. There is no visible logo. and it struggles on mobile.
- This PR adjusts the navbar styling so that a logo is visible. We have also created a new essential travel navbar in patter library to see how this works (branch the same name)

#### What tests does this PR have?
None

#### How can this be tested?
Checkout this branch on both this repo `brands` and pattern library
in pattern library run `npm link ../brands` then run `npm start`
Navigate to organisms > Essential Navbar and check how this works across device

#### Screenshots / Screencast
screen shot 2016-02-25 at 17 12 46
screen shot 2016-02-25 at 17 13 03

#### What gif best describes how you feel about this work?
Blink 182

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

File: brands/essentialtravel/src/less/_navbar.less [hunk 1]
```diff
@@ -1,78 +1,52 @@
...
+    }
+  }
+}
 
-}
\ No newline at end of file
+@media (min-width: @screen-sm-min) {
+  .navbar-inverse {
+    min-height: @navbar-inverse-height;
+    font-size: 20px;
```
Comment (stuartgiles) [inline brands/essentialtravel/src/less/_navbar.less]: font-size: @font-size-large;

File: brands/essentialtravel/src/less/_navbar.less [hunk 2]
```diff
@@ -1,78 +1,52 @@
...
+.navbar-right {
+  margin-right: @grid-gutter-width/2;
 }
 
-#product-nav {
-	position:relative;
-	font-size: @font-size-large;
-	font-weight: 300;
-	margin-bottom: 20px;
-	.navbar-nav .active:after {
```
Comment (stuartgiles) [inline brands/essentialtravel/src/less/_navbar.less]: Why delete all this? It's not related to this PR.

I think I'd keep this stuff until you write a replacement.
1. .navbar-nav .active
2. .tab-content
3. .tab-pane

File: brands/essentialtravel/src/less/_navbar.less [hunk 3]
```diff
@@ -1,78 +1,52 @@
...
 
-}
\ No newline at end of file
+@media (min-width: @screen-sm-min) {
+  .navbar-inverse {
+    min-height: @navbar-inverse-height;
+    font-size: 20px;
+    .navbar-nav {
+      & > li > a {
+        padding-top:28px;
```
Comment (stuartgiles) [inline brands/essentialtravel/src/less/_navbar.less]: Could you make use of a bootstrap paddingding variable?
Something like @padding-large-vertical:

Comment (jodiedoubleday) [inline brands/essentialtravel/src/less/_navbar.less]: There is no useful variable to use, checked them all.

**Comment by elliottcrush** [comment]
Good with me :one: :+1:

**Comment by stuartgiles** [comment]
Thanks for the tweaks.
Happy :+1:

**Pull request merged**